packages/react-icons/bin/index.js

```diff
@@ -48,7 +45,14 @@ const convertVariant = async (file) => {
             {componentName}
         );
         return Promise.all([
-            fs.appendFile(`./generated/${iconName}/index.ts`, `export * from './${variantName}';\n`),
+            fs.appendFile(
+                './generated/index.ts',
+                `export const ${componentName} = loadable(() => import('./${iconName}/${variantName}'));\n`
+            ),
+            fs.appendFile(
+                `./generated/${iconName}/index.ts`,
+                `export {default as ${componentName}} from './${variantName}';\n`
+            ),
```

Comment:
I this really needed?

Reply:
It was just to avoid breaking the existing imports if ever they were used